flink-connectors/flink-connector-kafka-0.8/src/main/java/org/apache/flink/streaming/connectors/kafka/Kafka08JsonTableSink.java

```diff
@@ -42,7 +43,7 @@ public Kafka08JsonTableSink(String topic, Properties properties, FlinkKafkaParti
 	}
 
 	/**
-	 * Creates {@link KafkaTableSink} for Kafka 0.8
+	 * Creates {@link KafkaTableSink} for Kafka 0.8 .
```

Comment:
Okay, it was clearly a pattern but wasn't sure if it was an unexpected programmatic replacement. I think it's find leaving as is but expect that someone will eventually flag it for fixing.